hw-model/src/lib.rs

```diff
@@ -109,6 +109,8 @@ impl TrngMode {
     }
 }
 
+const EXPECTED_CALIPTRA_BOOT_TIME_IN_CYCLES: u64 = 20_000_000; // 20 million cycles
```

Comment:
This value is way too short: since adding all the pairwise-consistency tests, the smoke_tests are currently taking 32 million cycles, and they don't even use LMS.

Reply:
FYI: Running nightly workflow here:

https://github.com/chipsalliance/caliptra-sw/actions/runs/6151518010

Reply:
Looks like LMS upper bound is around 4.5M cycles, so we can probably set this to 38 million, or maybe 40 if you want a nice round number.